hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFSWithObjectStoreCreate.java

```diff
@@ -303,6 +304,12 @@ public void testMPUFailDuetoDirectoryCreationBeforeComplete()
 
   }
 
+  @Test(expected = FileAlreadyExistsException.class)
```

Comment:
Thanks for the additional test. Can we also add a test case that exercises the REST API to create a file and dir with the same name, in both orders?

Reply:
You mean use OzoneBucket or add a robot test using S3.
If robot test, we don't have test suite with this flag enabled, opened a jira 
https://issues.apache.org/jira/browse/HDDS-4154

Reply:
Added a test